pysages/backends/hoomd.py

```diff
@@ -125,15 +124,24 @@ def bias(snapshot, state, sync_backend):
 
 
 def bind(context, sampling_method, **kwargs):
-    #
+    """
+    Bind pysages to hoomd-blue.
+
+    context -- hoomd simulation context
+    sampling_method -- pysages sampling_method
+    callback -- callback method with call signature `callback(snapshot, state)`
+      called after PySages updated. Example: logging of CVs.
+    """
     helpers, bias = build_helpers(context)
-    #
+
     wrapped_context = ContextWrapper(context)
+
     snapshot = take_snapshot(wrapped_context)
     method_bundle = sampling_method(snapshot, helpers)
-    sync_and_bias = partial(bias, sync_backend = wrapped_context.synchronize)
-    #
-    sampler = Sampler(method_bundle, sync_and_bias)
+    sync_and_bias = partial(bias, sync_backend=wrapped_context.synchronize)
+    pysages_callback = kwargs.get("callback", None)
```

Comment:
If we make callback an explicit argument then this won't be needed.

Reply:
sure, we can make it an explicit argument.
I didn't do it, because it more typing, but if improves the design, I'll change it.

Reply:
fixed